src/cli.rs

```diff
@@ -74,6 +74,26 @@ pub struct Opts {
     #[arg(long, overrides_with = "no_ignore_vcs", hide = true, action = ArgAction::SetTrue)]
     ignore_vcs: (),
 
+    /// Do not require a git repository to respect gitignores.
+    #[arg(
+        long,
+        overrides_with = "require_git",
+        hide_short_help = true,
+        // same description as ripgrep's flag
+        long_help = "By default, fd will only respect global gitignore rules, .gitignore rules,\
+                     and local exclude rules if fd detects that you are searching inside a\
+                     git repository. This flag allows you to relax this restriction such that\
+                     fd will respect all git related ignore rules regardless of whether you're\
+                     searching in a git repository or not.\n\n\
+                     This flag behaves the same as --no-require-git flag from ripgrep.\n\n\
+                     This flag can be disabled with --require-git."
```

Comment:
FYI this is the same description as `--no-require-git` for ripgrep https://github.com/BurntSushi/ripgrep/blob/3bb71b0cb8727ac43237af78ba5c707298de0128/crates/core/app.rs#L2216-L2225

In your opinion does this port require a more formal attribution?

Reply:
fd isn't my project, but I personally wouldn't describe behavior by referencing other tools like this.

> In your opinion does this port require a more formal attribution?

On my end, no. It's probably good sense to mention it in a commit message or perhaps a source code comment so that others looking at it in the future have proper context, but I don't think there's any need for anything more.

Reply:
I did mention it in a source-code comment

Reply:
Thanks!  I did already mention this in a source code comment 👍